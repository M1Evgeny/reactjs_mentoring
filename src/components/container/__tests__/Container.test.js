import React from "react";
import { render } from "@testing-library/react";
import { Container } from "..";

describe("Container test", () => {
  test("Container snapshot", () => {
    const { asFragment } = render(<Container />);
    expect(asFragment()).toMatchSnapshot();
  });
});
