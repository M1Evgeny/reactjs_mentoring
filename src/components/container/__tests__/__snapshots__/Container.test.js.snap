// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container test Container snapshot 1`] = `
<DocumentFragment>
  <div
    class="header-container"
  />
</DocumentFragment>
`;
